pkg/operation/botanist/component/etcd/monitoring.go

```diff
@@ -115,6 +115,18 @@ const (
         within the last hour.
       summary: High number of failed etcd proposals
 
+  - alert: KubeEtcd3HighMemoryConsumption
+    expr: sum(container_memory_working_set_bytes{pod="etcd-main-0",container="etcd"}) / sum(vpa_spec_container_resource_policy_allowed{allowed="max",container="etcd", targetName="etcd-main", resource="memory"}) > .5
```

Comment:
Conceptually, gardener no longer knows the etcd statefulsets - it only knows the `etcd.druid.gardener.cloud/etcd-(events|main)`. 

This said, I think these alerts should be migrated to the [etcd-druid](https://github.com/gardener/etcd-druid/) controller following the [monitoring extensibility](https://github.com/gardener/gardener/blob/master/docs/extensions/logging-and-monitoring.md#extensions-monitoring-integration) contract

Reply:
Generally, I agree. 
For this concrete PR, though, I guess we can proceed as there are already other alerts and rules that should be moved to g/etcd-druid.

Reply:
I have opened https://github.com/gardener/etcd-druid/issues/172 to track the migration of monitoring config into `etcd-druid`. But I suspect it might be picked only after multi-node is done.